packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -86,6 +86,8 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
   className?: string;
   /**
    * The properties that will get passed to the Suggestions component.
+   *
+   * TODO (adjective-object) remove the '| any' before the next major version
```

Comment:
Is this comment still relevant?

Reply:
hmm, thought I cleaned this up but I guess I forgot to push